wordpress-mobile/WordPressMocks
# Add project ruby dependency management

As a follow-up for #1, this adds the same kind of bundle configuration that the rest of WP projects use.

**To Test:**
- Run `bundle install && bundle exec pod --version`. It should complete without errors.
